Cython/Compiler/FusedNode.py

```diff
@@ -579,6 +580,12 @@ def make_fused_cpdef(self, orig_py_func, env, is_def):
         decl_code = Code.PyxCodeWriter(context=context)
         decl_code.put_chunk(
             u"""
+                cdef union {{lecheck_cname}}:
+                    unsigned int i
```

Comment:
Problem is, C int isn't guaranteed to be 4 bytes. And then, stdint.h isn't available in some MSVC versions...
I think we should provide the endianess check globally for the complete Cython module, i.e. do it in C and move it to ModuleSetupCode.c. There is an example for defining uint16_t in TypeConversion.c, uint32_t could be defined similarly. (Although ISTM that we should use a more safely escaped name than uint16_t there as well...)